### PR TITLE
refactor runtime_env proto

### PR DESCRIPTION
**Pull request by runzexia**

### CODE REVIEW

File: api/runtime_env.proto [hunk 1]
```diff
@@ -93,7 +93,7 @@ message RuntimeEnvCredential{
 	google.protobuf.StringValue description = 3;
 	map <string, string> content = 4;
 	google.protobuf.StringValue owner = 5;
-	repeated string runtime_env_id = 6;
+	repeated string runtime_env_ids = 6;
```
Comment (zheng1) [inline api/runtime_env.proto]: This field should be `runtime_env_id`

File: pkg/cmd/api/spec/index.html [hunk 2]
```diff
@@ -1,97 +1,97 @@
...
-    layout: "StandaloneLayout"
-  })
-
-  window.ui = ui
-}
-</script>
-</body>
-
-</html>
+<!-- HTML for static distribution bundle build -->
```
Comment (zheng1) [inline pkg/cmd/api/spec/index.html]: Why we need modify this file?

File: api/runtime_env.proto [hunk 3]
```diff
@@ -58,13 +58,13 @@ message DescribeRuntimeEnvsRequest{
 	google.protobuf.StringValue search_word = 3;
 	google.protobuf.StringValue selector = 4;
 	repeated string owner = 5;
-	google.protobuf.UInt32Value limit = 6;
-	google.protobuf.UInt32Value offset = 7;
-	google.protobuf.UInt32Value verbose = 8;
+	uint32 limit = 6;
+	uint32 offset = 7;
+	uint32 verbose = 8;
```
Comment (zheng1) [inline api/runtime_env.proto]: There is no comparison to use `verbose` here.

File: api/runtime_env.proto [hunk 4]
```diff
@@ -114,13 +114,13 @@ message DescribeRuntimeEnvCredentialsRequset{
 	repeated string status = 2;
 	google.protobuf.StringValue search_word = 3;
 	repeated string owner = 4;
-	google.protobuf.UInt32Value limit = 5;
-	google.protobuf.UInt32Value offset = 6;
-	google.protobuf.UInt32Value verbose = 7;
+	uint32 limit = 5;
+	uint32 offset = 6;
+	uint32 verbose = 7;
```
Comment (zheng1) [inline api/runtime_env.proto]: There is no comparison to use `verbose` here.

**Comment by runzexia** [comment]
done

File: api/runtime_env.proto (hunk 3 above)
Comment (runzexia) [reply api/runtime_env.proto]: One question is whether to return runtime_env_credential info when I query runtime_env.Or do I just return all the info?

Comment (zheng1) [reply api/runtime_env.proto]: We return runtime_env_credential by default, and the first phase does not seem necessary to add `verbose`.

**Comment by runzexia** [comment]
Because Protogen swagger has some bugs in the HTTP GET request, the string type is temporarily used to pass in the label param.
https://github.com/grpc-ecosystem/grpc-gateway/issues/578

**Comment by runzexia** [comment]
done

**Comment by runzexia** [comment]
done.

**Comment by zheng1** [comment]
```diff
diff --git a/api/runtime_env.proto b/api/runtime_env.proto
index f86c8ea..434a1e9 100644
--- a/api/runtime_env.proto
+++ b/api/runtime_env.proto
@@ -28,13 +28,20 @@ option (grpc.gateway.protoc_gen_swagger.options.openapiv2_swagger) = {
 	produces: "application/json";
 };
 
+message RuntimeEnvLabel {
+	google.protobuf.StringValue runtime_env_label_id = 1;
+	google.protobuf.StringValue runtime_env_id = 2;
+	google.protobuf.StringValue label_key = 3;
+	google.protobuf.StringValue label_value = 4;
+}
+
 message RuntimeEnv{
 	google.protobuf.StringValue runtime_env_id = 1;
 	google.protobuf.StringValue name = 2;
 	google.protobuf.StringValue description = 3;
 	google.protobuf.StringValue runtime_env_url = 4;
 	google.protobuf.StringValue runtime_env_credential_id = 5;
-	google.protobuf.StringValue labels = 6;
+	repeated RuntimeEnvLabel labels = 6;
 	google.protobuf.StringValue owner = 7;
 	google.protobuf.StringValue status = 8;
 	google.protobuf.Timestamp create_time = 9;

```
Please apply the patch above, this cannot be deleted.

**Comment by zheng1** [comment]
LGTM. Please squash the git commits with `git rebase -i`

**Comment by runzexia** [comment]
done

**Pull request merged**